casepro/backend/__init__.py

```diff
@@ -199,6 +199,15 @@ def fetch_contact_messages(self, org, contact, created_after, created_before):
                  Message.as_json() for incoming messages and Outgoing.as_json() for outgoing messages.
         """
 
+    @abstractmethod
+    def get_url_patterns(self):
+        """
+        Returns the list of URL patterns that should be registered for this
+        backend.
```

Comment:
Thus far we've had flake8 set with max-line-length=120, so no need to wrap to 79 (?)

I've tried to keep the code style to basically Django + 120 char wide lines.